bundles/core/src/test/resources/carousel/exporter-carousel1.json

```diff
@@ -3,6 +3,7 @@
     "autoplay": true,
     "delay": 7000,
     "autopauseDisabled": true,
+    "controlsPrepended": false,
```

Comment:
It would great to have a test with it `true` as well

Reply:
added in 15801c2b456ec2ea339764304247514deb56967c